srcpkgs/firejail/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'firejail'
 pkgname=firejail
-version=0.9.56
-revision=2
+version=0.9.58
+revision=1
 build_style=gnu-configure
 configure_args="--enable-apparmor"
+conf_files="/etc/firejail/* /etc/apparmor.d/local/firejail-local"
 makedepends="libapparmor-devel"
 short_desc="SUID security sandbox program"
 maintainer="Duncaen <duncaen@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://firejail.wordpress.com"
-#changelog="https://raw.githubusercontent.com/netblue30/firejail/master/RELNOTES"
+changelog="https://raw.githubusercontent.com/netblue30/firejail/${version}/RELNOTES"
```

Comment:
`changelog` is still not a real field and I prefer to keep it at master.

Reply:
Sure, restored.